agave/filters.py

```diff
@@ -20,3 +23,68 @@ def generic_query(query: QueryParams) -> Q:
     if 'count' in fields:
         del fields['count']
     return filters & Q(**fields)
+
+
+def get_by_id(cls, id: str):
```

Comment:
Esta funcion tambien puede agregarse dentro de cada modelo, por ejemplo:

```
class MongoModel(BaseModel, Document):
    meta = {'allow_inheritance': True}

    def dict(self) -> DictStrAny:
        return self._dict(mongo_to_dict)

    def get_by_id(cls, id: str):
        try:
            id_obj = cls.model.objects.get(id=id)
        except Exception as exc:
            if isinstance(exc, DoesNotExist):
                raise
        return id_obj

```

Pero al momento de implementar en rest-api se ve un poco extrano:
 `model = cls.model.get_by_id(cls, id=id)`

Reply:
Why not just name it `get()` and allow for more `user_id` to also be passed in?

Reply:
Do you mean use `cls.model.get(...)` instance of `cls.model.get_by_id()`?

What I don't catch is the way to implement `model = cls.model.get(cls, id=id)` where `cls` is used twice. 

The `user_id` is gotten from `cls` in the following way `user_id=cls.current_user_id`.

How would you be the definition of the `get` function?